.circleci/config.yml

```diff
@@ -769,294 +769,302 @@ jobs:
           when: always
       - store_artifacts:
           path: ../uitest_android_results
-  api_1-amplify_e2e_tests:
+  layer-amplify_e2e_tests:
```

Comment:
I think the changes to this file need to be reverted

Reply:
The file gets processed pre commit, can't really help it.